charts/seed-terraformer/charts/packet-infra/values.yaml

```diff
@@ -0,0 +1,4 @@
+projectID: abcd-5678
+clusterName: test-namespace
+sshPublicKey: sshkey-12345
+
```

Comment:
I think you need https://github.com/gardener/gardener/blob/master/charts/seed-terraformer/charts/aws-infra/values.yaml#L17-L22 here as well.

Reply:
Adding.